script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -0,0 +1,89 @@
+; Script generated by the Inno Script Studio Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Git LFS"
+#define MyAppVersion "1.1.0.0"
+#define MyAppPublisher "GitHub, Inc"
+#define MyAppURL "https://git-lfs.github.com/"
+#define MyAppExeName "Git-LFS-Installer.exe"
```

Comment:
Instead of defining this here, can we make it dynamic based on the version?  `git-lfs-windows-amd64-{#MyAppVersion}.exe`. Also, does the version need 4 digits? If we can get away with a `MyAppVersion` of `x.x.x`, I think I'd prefer that.